test/e2e/postinstall/machinesets/infra_test.go

```diff
@@ -304,7 +304,7 @@ func TestAutoscalingMachinePool(t *testing.T) {
 			Name:      "busybox",
 		},
 		Spec: appsv1.DeploymentSpec{
-			Replicas: pointer.Int32(100),
+			Replicas: pointer.Int32(120),
```

Comment:
we don't still need this delta, do we?

Reply:
[Done](https://github.com/openshift/hive/compare/197a73ccb4f9b9fdd60c60a241c4e597a85e47f4..58740ecacecf162721b337be3a1a67c11ea3578a)